### ISSUE TITLE
Error: Invalid glob import syntax: Expect CallExpression, got SequenceExpression

### ISSUE THREAD

**Issue by jrson83** [antfu/vite-plugin-glob#21]
I try to get the plugin working with a fresh clean install of `vite-plugin-ssr` [preact-client-routing-example](https://github.com/brillout/vite-plugin-ssr/tree/master/examples/preact-client-routing). Please can you give a helping hand how to get this work. As soon as I add `GlobPlugin()` to plugins, it's throwing the error. I have setup a repo for reproduction:

https://github.com/jrson83/vite-plugin-glob-preact-example

```js
Error: Invalid glob import syntax: Expect CallExpression, got SequenceExpression
    at err (D:\______jrson.de\preact-client-routing\node_modules\vite-plugin-glob\dist\index.cjs:97:17)
    at D:\______jrson.de\preact-client-routing\node_modules\vite-plugin-glob\dist\index.cjs:115:13
    at Array.map (<anonymous>)
    at parseImportGlob (D:\______jrson.de\preact-client-routing\node_modules\vite-plugin-glob\dist\index.cjs:93:24)
    at transform (D:\______jrson.de\preact-client-routing\node_modules\vite-plugin-glob\dist\index.cjs:208:23)
    at TransformContext.transform (D:\______jrson.de\preact-client-routing\node_modules\vite-plugin-glob\dist\index.cjs:72:28)
    at Object.transform (D:\______jrson.de\preact-client-routing\node_modules\vite\dist\node\chunks\dep-59dc6e00.js:38900:53)
    at async doTransform (D:\______jrson.de\preact-client-routing\node_modules\vite\dist\node\chunks\dep-59dc6e00.js:55857:29
```

**Comment by brillout** [comment]
I'll have a look, thanks for the ping.

**Comment by brillout** [comment]
Done: https://github.com/antfu/vite-plugin-glob/pull/22.

**Comment by antfu** [comment]
Fixed in #22
